Directory.Build.props

```diff
@@ -5,13 +5,13 @@
     <TargetDotnetVersions>net6.0;$(DefaultTargetDotnetVersion)</TargetDotnetVersions>
     <Platforms>x64;ARM64</Platforms>
     <AvaloniaVersion>0.10.17</AvaloniaVersion>
-    <CefGlueVersion>120.6099.3</CefGlueVersion>
+    <CefGlueVersion>120.6099.200</CefGlueVersion>
   </PropertyGroup>
 
   <PropertyGroup>
     <AssemblyVersion>2.0.0.0</AssemblyVersion>
     <FileVersion>2.0.0.0</FileVersion>
-    <Version>2.121.2</Version>
+    <Version>3.120.0</Version>
```

Comment:
@joaompneves we put back the 120 since we are using this CEF version - it was wrongly set to 121 in [this commit](https://github.com/OutSystems/WebView/commit/40582fe5d35cb7357ff1ef66eaf28d83a0c2585e)